src/app/handlers/counting.handler.ts

```diff
@@ -38,6 +39,9 @@ export class CountingHandler extends Handler {
         await message.react(this._buzzEmoji);
       }
 
+      // Increment count in counting leaderboard
+      const userDoc = await this._getUserCountingDoc(message.author.id);
+      await CountingLeaderboardModel.updateOne(userDoc, { $inc: { count: 1 } });
```

Comment:
is there no updateOrInsert type thing? ?

Reply:
I tried, but I don't think I can do it with an insert too